WordPressKit.podspec

```
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressKit'
-  s.version       = '8.4.0'
+  s.version       = '8.4.0-beta.1'
```

It looks like `8.4.0` is about to be released (#614) so we'd want to bump this up to the `8.5.0` beta:
```suggestion
  s.version       = '8.5.0-beta.1'
```